recipe-server/client/control_new/components/common/LoadingOverlay.js

```diff
@@ -14,16 +14,30 @@ import { areAnyRequestsInProgress } from 'control_new/state/requests/selectors';
 export default class LoadingOverlay extends React.Component {
   static propTypes = {
     children: PropTypes.any,
+    condition: PropTypes.bool,
     loading: PropTypes.bool.isRequired,
+    useCondition: PropTypes.bool,
   };
 
   static defaultProps = {
     children: null,
+    condition: false,
```

Comment:
can't we change this to `null` and then we don't need `useCondition`.

we would just do this:
```javascript
const isCurrentlyLoading = condition === null ? loading : condition;
```